### ISSUE TITLE
When is the new release comming?

### ISSUE THREAD

**Issue by moshpirit** [notion-enhancer/desktop#789]
Last update is from 31 Dec, 2021. During this year, many bugs seem to have been fixed right now. Is it save for us to compile it directly from git or isn't it stable enough? Should we expect some new release during the coming weeks?

**Comment by dragonwocky** [comment]
> Last update is from 31 Dec, 2021.

The notion-enhancer is a side project that I work on when I have time to- unless somehow I start getting a significant income from it, that's how it will always be. I was pretty busy for most of 2022 and haven't have time to work on the notion-enhancer until the last couple of months.

> Is it save for us to compile it directly from git or isn't it stable enough?

Feel free to try, if you would like. The core that most extensions and themes depend upon has been significantly improved, though it still isn't 100% done yet. The menu is about half-done. One theme is available. None of the other extensions have been updated to work with the new core yet. So... it's stable, sure. Just won't have many/any features.


> Should we expect some new release during the coming weeks?

It's coming when it's ready. I plan to release it as soon as the core and menu are complete and a few themes/extensions have been updated, with the rest of them coming in subsequent updates. I can't give any guaranteed dates, though, because I don't have any.

**Comment by moshpirit** [comment]
Sorry, it wasn't my intention to bother you. I can't imagine how many hours this kind of project might take.

**Comment by dragonwocky** [comment]
@moshpirit You're all right, sorry if I came off as a little irritated! I just get this question a lot. The update is coming soon, I just don't know exactly when - if you haven't already you may like to join the notion-enhancer Discord server, I post progress updates fairly frequently in the #announcements and #development channels there: https://discord.gg/sFWPXtA

**Comment by MichaelToLearn** [comment]
> @moshpirit You're all right, sorry if I came off as a little irritated! I just get this question a lot. The update is coming soon, I just don't know exactly when - if you haven't already you may like to join the notion-enhancer Discord server, I post progress updates fairly frequently in the #announcements and #development channels there: https://discord.gg/sFWPXtA

The link has been invalid.

**Comment by dragonwocky** [comment]
> The link has been invalid.

@MichaelToLearn hm, odd. It's definitely still working for me. You can try this one: https://discord.gg/zRCBD2QW
